### ISSUE TITLE
报错：Install failed: Execute command:

### ISSUE THREAD

**Issue by aplus42** [JettChenT/wechat-exporter#2]
(base) zj@appledeMacBook-Pro Downloads % sudo ./wechattweak-cli install
------ Download ------
------ Unzip ------
Execute command: rm -rf /Applications/WeChat.app/Contents/MacOS/WeChatTweak.framework; unzip /tmp/WeChatTweak.zip -d /Applications/WeChat.app/Contents/MacOS
Install failed: Execute command: rm -rf /Applications/WeChat.app/Contents/MacOS/WeChatTweak.framework; unzip /tmp/WeChatTweak.zip -d /Applications/WeChat.app/Contents/MacOS failed: {
    NSAppleScriptErrorAppName = "wechattweak-cli";
    NSAppleScriptErrorBriefMessage = "checkdir:  cannot create extraction direct           No such file or directory";/MacOS
    NSAppleScriptErrorMessage = "checkdir:  cannot create extraction directory:            No such file or directory";S
    NSAppleScriptErrorNumber = 2;
    NSAppleScriptErrorRange = "NSRange: {0, 157}";
}

macOS:13.2.1 (22D68)

**Comment by JettChenT** [comment]
参考一下 [这个](https://github.com/sunnyyoung/WeChatTweak-macOS/issues/621) issue。
TLDR: 运行命令的终端要给完全磁盘访问权限和App 管理权限
